public/app/plugins/datasource/grafana-azure-monitor-datasource/api/routes.ts

```diff
@@ -36,3 +36,18 @@ export function getAppInsightsApiRoute(azureCloud: string): string {
       throw new Error('The cloud not supported.');
   }
 }
+
+export function getDeepLinkRoute(azureCloud: string): string {
```

Comment:
I would move this function to `credentials.ts` and call it `getAzurePortalUrl`.
This routes file is for plugin route definitions.